modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to the NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
+
+.Procedure
+. Change the NODEPORT variable to the host port number assigned to your kubernetes NodePort service 
++
+[source,terminal]
+----
+$ export NODEPORT=30700
+----
+. Change the INTERFACE_IP value to the IP address from the host interface that you want to block. 
++
+[source,terminal]
+----
+$ export INTERFACE_IP=192.168.150.33
```

Comment:
I think they wanted to show the exact IP becuase its referenced later in the doc. And I think the sentence shows that it can be changed. Maybe I can add a `For example` at the end of the sentence to show that its a sample and that the user is meant to change themselves.

Reply:
Makes sense. Thank you for clarifying.